bootstrap/templates/kubernetes/apps/storage/openebs/ks.yaml.j2

```
@@ -2,19 +2,19 @@
 apiVersion: kustomize.toolkit.fluxcd.io/v1
 kind: Kustomization
 metadata:
-  name: &app local-path-provisioner
+  name: &app openebs
   namespace: flux-system
 spec:
   targetNamespace: storage
   commonMetadata:
     labels:
       app.kubernetes.io/name: *app
-  path: ./kubernetes/apps/storage/local-path-provisioner/app
+  path: ./kubernetes/apps/kube-system/openebs/app
   prune: true
   sourceRef:
     kind: GitRepository
     name: home-kubernetes
-  wait: false
+  wait: true
```

```suggestion
  wait: false
```

We don't need `wait: true` here since applications can use a HelmRelease `dependsOn` instead of a Flux Kustomization `dependsOn`